docs/_docs/plugins.md

```diff
@@ -910,7 +877,7 @@ You can find a few useful plugins at the following locations:
   plugin to generate publication lists from [BibSonomy](https://www.bibsonomy.org/).
 - [github-cards](https://github.com/edward-shen/github-cards): Creates styleable Github cards for your Github projects.
 - [disqus-for-jekyll](https://github.com/kacperduras/disqus-for-jekyll): A Jekyll plugin to view the comments powered by Disqus.
-- [jekyll-html](https://github.com/kacperduras/jekyll-html): A Jekyll plugin to use HTML tags in Jekyll pages, posts and collections. 
+- [jekyll-html](https://github.com/kacperduras/jekyll-html): A Jekyll plugin to use HTML tags in Jekyll pages, posts and collections.
```

Comment:
Can we avoid modifying this line (because history/blame)?

(I feel the pain of automatically trimming whitespace 😁)

Reply:
I configured Atom to fix this automatically, that page should be history soon once we ship a decent version of https://jekyll.github.io/directory/

Reply:
done